custom-schemes.md

```diff
@@ -2,40 +2,54 @@
 
 Existing specifications like [OpenID4VP](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-wallet-invocation) and [ISO 18013-7](https://www.iso.org/standard/82772.html) currently define the use of custom URL schemes (eg. `openid4vp:` and `mdoc:`) to enable verifier websites to invoke holder (wallet) applications. This approach works in some scenarios, but has a number of problems we aim to solve by designing a browser API which can replace the use of custom URL schemes in these specifications. This DRAFT document aims to outline the questions and concerns with the use of custom URL schemes for identity presentment in order to provide more detail on potential benefits of a dedicated browser API.
 
-1. Can wallets reliably determine their invoker?
+## Can wallets reliably determine their invoker?
 
-   - Android and iOS offer no facility to determine the web origin which triggered a navigation to a custom URL scheme. 
+   - Android and iOS offer no facility to determine the web origin which triggered a navigation to a custom URL scheme.
    - This seems to increase the opportunity for attacks such as replay attacks, man-in-the-middle attacks and phishing. It also seems to limit the opportunity for abuse detection and prevention.
    - A browser API can securely pass the requesting origin on to the wallet so that the wallet can know with confidence what origin the user was on when the request was initiated.
+   
+## How are wallet apps supposed to pass info back to the invoking page?
 
-2. Can wallets limit requests to secure contexts?
+Does the page give the wallet a URL to navigate back to as part of the custom scheme URL it uses to launch the wallet?
 
-   - If a user uses a website via http (not https) and it invokes a custom scheme, it's possible that an attacker with access to the network could eavesdrop on or even modify the request (such as by replacing the verifier's public key with their own). 
-   - A browser API could take the position that it will function only within the context of a secure context (https).
+If so, that's a problem because it won't go back to the same tab. Apps don't have the ability to return information to the same page that invoked them after a custom scheme invocation; they can just open a new page in a new tab.
```

Comment:
```suggestion
  - If so, that's a problem because it won't go back to the same tab. Apps don't have the ability to return information to the same page that invoked them after a custom scheme invocation; they can just open a new page in a new tab.
```